File: .ProgramingBasics/.SimpleConditionalStatements/03.EvenOrOdd/src/EvenOrOdd.java
import java.util.Scanner;

public class EvenOrOdd {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num = Integer.parseInt(scanner.nextLine());

        if (num % 2 == 0) {
            System.out.println("even");
        }
        else {
            System.out.println("odd");
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/03.EvenOrOdd/03.EvenOrOdd/EvenOrOdd.cs
using System;

namespace _03.EvenOrOdd
{
    public class EvenOrOdd
    {
        public static void Main(string[] args)
        {
            var num = int.Parse(Console.ReadLine());

            if (num % 2 == 0)
            {
                Console.WriteLine("even");
            }
            else
            {
                Console.WriteLine("odd");
            }
        }
    }
}
